src/pages/arbeidsgiver/[narmestelederid]/[oppfolgingsdialogId]/index.tsx

```diff
@@ -41,7 +46,14 @@ const Content = ({
       );
     }
     case "MOTTATTFLEREGODKJENNINGER": {
-      return <div>MOTTATTFLEREGODKJENNINGER</div>;
+      return (
+        <GodkjennPlanAvslattOgGodkjent
+          oppfolgingsplan={oppfolgingsplan}
+          description={`${oppfolgingsplan?.arbeidstaker?.navn} har gjort noen
+        endringer i planen og sendt den tilbake til deg.`}
+          altinnTargetAudience={"arbeidsgiveren din"}
```

Comment:
🤔 Skal det ikke være `arbeidstakeren` her?

Reply:
Jeg kjenner litt på at det er vanskelig å skjønne hva `altinnTargetAudience` betyr, måtte gå inn i komponentene for å sjekke hva det ble brukt til. Kanskje bruke `motpart` e.l. her også?

Reply:
Enig, kan endre :)